powerauth-webflow/package.json

```diff
@@ -36,7 +36,8 @@
     "react-intl-redux": "^0.5.0",
     "hard-source-webpack-plugin": "^0.5.0-rc.2",
     "bluebird": "^3.5.0",
-    "intl": "^1.2.5"
+    "intl": "^1.2.5",
+    "sanitize-html": "^1.20.1"
```

Comment:
We should probably do some HTML sanitization server side as well...

Reply:
Ok, I found this sanitizer: https://github.com/OWASP/java-html-sanitizer, we can try it.